config/variables/common.yml

```diff
@@ -10,7 +10,7 @@
 variables:
 
   deploymentRegion: canadacentral
-  serviceConnection: spn-azure-platform-ops
```

Comment:
Can you reset this variable to the original version?

```
serviceConnection: spn-azure-platform-ops
```

Reply:
Ready for review!